packages/peregrine/lib/store/actions/cart/asyncActions.js

```diff
@@ -499,23 +497,25 @@ function toRESTCartItem(cartId, payload) {
     return cartItem;
 }
 
-export async function writeImageToCache(item = {}) {
-    const { media_gallery_entries: media, sku } = item;
+export const writeImageToCache = (item = {}) =>
+    async function thunk() {
```

Comment:
To make this useable in the talon from the cart context I had to update the signature of this function otherwise an error was thrown about not using plain objects or something. This is likely a bug with how we connect the actions but I'm not willing to solve it here because we're likely to throw away this code when we move `getCartDetails` to graphql. At that time we will just get the image url from the GQL response.